shoov/modules/custom/shoov_ci_build/handlers/shoov_ci_build_trigger_button.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Contains shoov_ci_build_trigger_button.
+ */
+
+class shoov_ci_build_trigger_button extends views_handler_field_entity {
```

Comment:
I used word "button" because for other links (edit, delete, view) it's also called "button". Change to `shoov_ci_build_send_to_server_link`?